karton/core/task.py

```diff
@@ -261,32 +263,42 @@ def walk_payload_bags(self) -> Iterator[Tuple[Dict[str, Any], str, Any]]:
             for key, value in payload_bag.items():
                 yield payload_bag, key, value
 
-    def iterate_resources(self) -> Iterator[Tuple[str, ResourceBase]]:
+    def walk_payload_values(self) -> Iterator[Tuple[str, Any]]:
```

Comment:
Should we maybe call this `walk_payload_items`? So we have a similar terminology to `dict.items()`

Reply:
Agree, done!